modules/gitrepo/command.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2025 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package gitrepo
+
+import (
+	"context"
+
+	"code.gitea.io/gitea/modules/git/gitcmd"
+)
+
+func runCmdString(ctx context.Context, repo Repository, cmd *gitcmd.Command) (string, string, error) { //nolint:unparam // the second return parameter maybe used in the future
```

Comment:
maybe remove the second return since it can be added in the future and go will let us know where it is used?

Reply:
ec62dfb256